lms/static/js/staff_debug_actions.js

```diff
@@ -67,7 +67,8 @@ var StaffDebug = (function(){
                 {text: text},
                 {interpolate: /\{(.+?)\}/g}
             )
```

Comment:
Missing semi-colon (on line 69). This file has a ton of missing semicolons-- could you please clean it up while you are editing the file?